packages/types/src/clerk.ts

```diff
@@ -312,6 +320,28 @@ export type RedirectOptions = {
   redirectUrl?: string | null;
 };
 
+export type SetActiveParams = {
+  /**
+   * The session resource or session id (string version) to be set as active.
+   * If `null`, the current session is deleted.
+   */
+  session?: ActiveSessionResource | string | null;
+
+  /**
+   * The organization id to be set as active in the current session.
+   * If `null`, the currently active organization is removed as active.
+   */
+  organization?: string | null;
```

Comment:
@alex-ntousias This should accept an `OrganizationResource | string | null` as in `session` argument;

Reply:
@SokratisVidros Initially I've had it like that. 
After a chat with Peter though, it became clear that it's better to just have it as id. It simplifies the code by avoiding certain if statements, and makes sure that the organization resource actually exists in the user's memberships (because we have to iterate through them to find it).

The downside is that the caller, if he has an organization resource, he will have to pass `organization.id`, which didn't seem like a big deal.
WDYT?

Reply:
I see. It's a good DX practise to accept both in the JS/TS world. Let's do it.